math/z_score.cpp

```diff
@@ -0,0 +1,72 @@
+#include <iostream>
+#include <vector>
+#include <cmath>
+
+
+bool Verification(std::vector<int>& data,int &x){
+    for(int i = 0;i <= data.size() - 1; i++){
+        if(x == data[i]){
+            return true;
+        }
+    }
+    return false;
+}
+
+void Z_Score(std::vector<int>& data,int &x){
+    //mean
+    int resultMean;
+    int Sum = 0;
+    for(int i = 0; i <= data.size() - 1;i++){
+        Sum += data[i];
+    }
+    resultMean = Sum / data.size();
+    std::cout << "Nilai rata-rata: " << resultMean << std::endl;
+
+    //standart deviation
+    int denominator = data.size() - 1;
+    int numerator;
+    float Standar_Dev;
+    float sum_numerator = 0;
+    for(int i = 0; i <= data.size() - 1;i++){
+        numerator = sqrt((data[i] - resultMean) * (data[i] - resultMean)); //sqrt(pow((data[i] - mean))); 
```

Comment:
![image](https://github.com/user-attachments/assets/3a81efc3-2dcd-4bee-8f62-8cbcfd1d8538)

```suggestion
        int numerator = sqrt((data[i] - resultMean) * (data[i] - resultMean)); //sqrt(pow((data[i] - mean))); 
```